nrf52-code/radio-app/src/lib.rs

```diff
@@ -1,10 +1,26 @@
 #![no_std]
 
-use panic_probe as _;
+use cortex_m_rt::exception;
+// use panic_probe as _;
```

Comment:
Do you want to keep this commented line?

Reply:
Good spot - I have reverted to using panic-probe.